src/drf_yasg/inspectors/field.py

```diff
@@ -465,11 +465,21 @@ def field_to_swagger_object(self, field, swagger_object_type, use_references, **
         return NotHandled
 
 
+class HiddenFieldInspector(FieldInspector):
+    """Hide ``HiddenField``."""
+
+    def field_to_swagger_object(self, field, swagger_object_type, use_references, **kwargs):
+        if isinstance(field, serializers.HiddenField):
+            return None
+        else:
```

Comment:
The ``else`` here is not needed, and could be removed to keep the style consistent with the other classes above.